linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -1090,62 +1088,45 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
    */
   def genTopLevelModuleExportDef(cd: LinkedClass,
       tree: TopLevelModuleExportDef): js.Tree = {
```

Comment:
I think you can inline this method above.

Reply:
Done